style/style.css

```diff
@@ -0,0 +1,218 @@
+    justify-content: flex-end;
+    text-align: center;
+    float: right;
+    height: 40px;
+    align-items: center;
+    font-size: 12px;
+}
+#up-info a {
+  text-decoration: none;
+  color:#0077c5;
+}
+#up-info button {
+  width: 105px;
+  height: 35px;
+  background-color: #0077c5;
+  color: white;
+  margin-left: 12px;
+  margin-right: 15px;
+  font-family: 'Roboto', sans-serif;
+  font-size: 15px;
+  border: 1px solid #333333;
+  border-radius: 5px;
+  cursor: pointer;
+}
+section {
+  margin-top: 35px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.central-section {
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    margin-top: 10px;
+    border: 1px solid #c7c7c7;
+    width: 380px;
+    height: 670px;
+    justify-self: center;
+    align-self: center;
+    text-align: center;
+}
+fieldset {
+  border: 0;
+  margin: 0;
+  padding: 0;
+}
+#form h3 {
+  margin-top: 25px;
+  margin-bottom: -15px;
+  font-size: 23px;
+  font-weight: normal;
+}
+#form p {
+  font-size: 14px;
+}
+#form a {
+  text-decoration: none;
+  font-weight: normal;
+  font-size: 12px;
+  color: #0077c5;
+}
+
+.form-info {
+  padding: 20px 20px 0 20px;
+}
+.form-info label p {
+  text-align: left;
+  margin-left: 10px;
+  margin-bottom: -10px;
+  font-size: 12px;
+  font-weight: normal;
+  color: black;
+  margin-bottom: 5px;
+}
+.form-info input {
+  background: #ffffff;
+  border: 1px solid #c6c7cc;
+  padding: 12px;
+  margin-top: 6px;
+  width: 300px;
+  margin-bottom: 30px;
+  border-radius: 3px;
+}
+.form-info input:hover {
+  box-shadow: 0 0 5px 0 rgba(106, 184, 230, 0.774);
+}
+.form-info #opened {
+  width: 323px;
+  height: 44px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border: 1px solid #c6c7cc;
+  border-radius: 3px;
+  margin-left: 8px;
+}
+.form-info #opened:hover {
+  box-shadow: 0 0 5px 0 rgba(106, 184, 230, 0.774);
+}
+.form-info #opened input {
+  border: none;
+  margin: none;
+  height: 19px;
+  margin: 0em;
+}
+.form-info #opened input:hover {
+}
+.form-info #opened img {
+  padding-right: 15px;
+  color: #c7c7c7;
+}
+.form-info #message p {
+  font-size: 10px;
+  margin-top: -25px;
+  margin-left: 8px;
+  margin-bottom: 30px;
+  text-align: left;
+}
+.form-info label {
+    color: #395870;
+    display: block;
+    font-weight: bold;
+    margin-bottom: 5px;
+}
+.form-info p {
+  margin-top: 0px;
+  margin-bottom: 20px;
+}
+
+#button-btn {
+  color: #ffffff;
+  background-color: #0077c5;
+  border-radius: 5px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  height: 47px;
+  width: 327px;
+  font-size: 17px;
+  cursor: pointer;
+  box-shadow: 0 0 1px 0 rgba(0, 0, 0, 0.3);
+
+}
+.form-info .form-button {
+  color: #ffffff;
+  background-color: #0077c5;
+  border-radius: 5px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  height: 47px;
+  width: 327px;
+  font-size: 17px;
+  cursor: pointer;
+  border: 1px solid #333333;
+}
```

Comment:
- [ ] These both appear to do the same thing. It is bad practice to repeat the same rules. You can consolidate these by adding 1 common class to the elements you want to be styled in this way.

- You can read more about good and bad CSS practices in [this article](https://speckyboy.com/good-bad-css-practices/).

![Screen Shot 2020-07-15 at 12 39 19 AM](https://user-images.githubusercontent.com/48804258/87483274-a0b3ef80-c633-11ea-8238-5a6b11ac0b9f.png)